SwiftBasicsExercise.playground/Pages/Dictionary_6.2.6.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,17 @@
+//MARK: Dictionary excercise 6
+var numbers = [1, 2, 3, 2, 3, 5, 2, 1, 3, 4, 2, 2, 2]
+let unique = Array(Set(numbers))
+var cnt = 0
+var freqOfNum: [Int: Int] = [:]
+unique.forEach { Items in
+    numbers.forEach { items1 in
+        if Items == items1 {
+            cnt+=1
+        }
+    }
+    freqOfNum[Items] = cnt
+    cnt = 0
+}
```

Comment:
We can do this in O(N) time
var numbers = [1, 2, 3, 2, 3, 5, 2, 1, 3, 4, 2, 2, 2]
var freq = Int: Int

numbers.forEach { number in
freq[number] = (freq[number] ?? 0) + 1
}

freq.keys.sorted().forEach { key in
print("(key) (freq[key] ?? 0)")
}

Solution of yours has O(N^2) complexity because of forEach inside forEach